crates/biome_grit_formatter/src/grit/lists/definition_list.rs

```diff
@@ -5,6 +5,22 @@ pub(crate) struct FormatGritDefinitionList;
 impl FormatRule<GritDefinitionList> for FormatGritDefinitionList {
     type Context = GritFormatContext;
     fn fmt(&self, node: &GritDefinitionList, f: &mut GritFormatter) -> FormatResult<()> {
-        format_verbatim_node(node.syntax()).fmt(f)
+        let mut join: JoinNodesBuilder<'_, '_, Line, GritFormatContext> =
+            f.join_nodes_with_hardline();
+
+        // TODO: Add separator
+        for definition in node {
+            match definition {
+                Ok(definition) => {
+                    join.entry(
+                        definition.syntax(),
+                        &format_or_verbatim(definition.format()),
+                    );
+                }
+                Err(_) => (),
+            }
+        }
```

Comment:
No need for using the `match`. You can simply do 

```rs
let definition = definition?;
```

Reply:
Thank you for this! Rust newbie here, so I appreciate this kind of input.